vignettes/data-filtering.Rmd

```diff
@@ -49,6 +56,7 @@ ggplot() +
   geom_point(data = mad_nwis, aes(x = Q, y = V), color = "red", alpha = .5)
 
 ```
+![cleaning1](/man/figures/cleaning1.png)
```

Comment:
This images will be created on render. Can you please removed these and the included images?